pepper-apis/src/main/resources/db-changes/seed/DDP-5102-add-arabic-portuguese-vietnamese.xml

```diff
@@ -0,0 +1,593 @@
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'MONTH_REQUIRED')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Bắt buộc phải điền vào trường tháng."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'NUM_OPTIONS_SELECTED')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Không đáp ứng được yêu cầu về lựa chọn tối thiểu hoặc tối đa."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'REGEX')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Không đáp ứng được yêu cầu về biểu thức chính quy của câu trả lời."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'REQUIRED')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Câu hỏi này cần được trả lời."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'YEAR_REQUIRED')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Bắt buộc phải điền vào trường năm."/>
+        </insert>
+
+
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'AGE_RANGE')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'fr')"/>
+            <column name="translation_text" value="L’âge doit être compris dans l’intervalle précisé."/>
+        </insert>
+        <update tableName="i18n_validation_msg_trans">
```

Comment:
Note: For these, I did not compare the translations with what's in the database. I simply took what we got from translators and plugged those in these updates.